src/client/v2/algod/healthCheck.ts

```diff
@@ -1,25 +1,15 @@
 import JSONRequest from '../jsonrequest.js';
-import IntDecoding from '../../../types/intDecoding.js';
+import { HTTPClientResponse } from '../../client.js';
 
 /**
  * healthCheck returns an empty object iff the node is running
  */
-export default class HealthCheck extends JSONRequest {
+export default class HealthCheck extends JSONRequest<void> {
   // eslint-disable-next-line class-methods-use-this
   path() {
     return '/health';
   }
 
-  async do(headers = {}) {
```

Comment:
I believe the reason for it was so that an error could be thrown if the request failed. But it wasn't really effective, since if `res.ok` was false the underlying `c.get` method would already throw an error.

And since this method doesn't return any data, the bigint decoding doesn't really make a difference.